firefox_private_network_privacy_notice/en-US.md

```diff
@@ -0,0 +1,42 @@
+## <span class="privacy-header-firefox">Firefox Private Network (Beta)</span> <span class="privacy-header-policy">Privacy Notice</span>
+Version 2.0, dated December 3, 2019
+{: datetime="2020-12-03" }
+
+## At Mozilla, we design products with your privacy in mind.
+
+This privacy notice explains what data is needed to offer Firefox Private Network (“Service”) and why.
+
+To learn more about how we receive and handle information, and how you can engage with your data, read the [Mozilla Privacy Policy](https://www.mozilla.org/en-US/privacy/).
```

Comment:
Remove the `en-US` in the URL. This will allow localized version to use the same link, even though this is not going to be localized for a while. People will be directed to the correct language (if translated) to the right one due to user browser language setting.